src/components/timeline/TrackIPC.css

```diff
@@ -0,0 +1,8 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.timelineMarkersIPC {
+  height: var(--markers-height, 25px);
```

Comment:
Seems like we don't need to repeat the 25px here, as it'll be easy to get out of date.